---
title: MSSQLSERVER_8630 | Microsoft Docs
ms.custom: ''
ms.date: 04/04/2017
ms.prod: sql
ms.reviewer: ''
ms.technology: supportability
ms.topic: language-reference
helpviewer_keywords:
- 8630 (Database Engine error)
ms.assetid: 45a9d084-b5ad-435b-ac19-5f4352515817
author: MashaMSFT
ms.author: mathoma
ms.openlocfilehash: 68361d48ce30a8e927182db97b9c2fd135aa9d4d
ms.sourcegitcommit: da88320c474c1c9124574f90d549c50ee3387b4c
ms.translationtype: HT
ms.contentlocale: ja-JP
ms.lasthandoff: 07/01/2020
ms.locfileid: "85727465"
---
# <a name="mssqlserver_8630"></a>MSSQLSERVER_8630
 [!INCLUDE [SQL Server](../../includes/applies-to-version/sqlserver.md)]
  
## <a name="details"></a>詳細  
  
| 属性 | 値 |  
| :-------- | :---- |  
|製品名|SQL Server|  
|イベント ID|8630|  
|イベント ソース|MSSQLSERVER|  
|コンポーネント|SQLEngine|  
|シンボル名|EXECUTION_ERR|  
|メッセージ テキスト|内部クエリ プロセッサ エラー:クエリ プロセッサ実行中に予期しないエラーが発生しました。|  
  
## <a name="explanation"></a>説明  
クエリ処理中に内部エラーが発生しました。  
  
## <a name="user-action"></a>ユーザーの操作  
主要なサービス プロバイダーに連絡し、 このエラー メッセージの情報を提供します。  
  
